### PR TITLE
[WIP] consolidate timecode information, update syntax

### PR DESCRIPTION
**Pull request by ablwr**
Moves and updates all timecode-related information to bottom of notes.md.

### CODE REVIEW

**Comment by ablwr** [comment]
This is good to review but should be reviewed after other related PRs are merged in, which might help with the diff looking a little better.

File: notes.md [hunk 1]
```diff
@@ -129,114 +117,128 @@ As an example a file named `intro.mkv` could have a `SegmentUID` of `0xb16a58609
...
+Because the NTSC track is at the original speed, it will used as the default value of 1.0 for its `TrackTimecodeScale`. The video will also be aligned to the NTSC track with the default value of 1.0.
 
-In the case of TrackJoinBlocks, the Blocks (from BlockGroup and SimpleBlock) of all the tracks SHOULD be used as if they were defined for this new virtual Track. When 2 Blocks have overlapping start or end timecodes, it's up to the underlying system to either drop some of these frames or render them the way they overlap. In the end this situation SHOULD be avoided when creating such tracks as you can never be sure of the end result on different platforms.
+The `TrackTimecodeScale` value to use for the PAL track would be calculated by determining how much faster the PAL track is than the NTSC track. In this case, because we know the video for the NTSC audio is being played back at 24fps and the video for the PAL audio is being played back at 25fps, the calculation would be:
+
+    (25 / 24) = ~ 1.04166666666666666667
```
Comment (retokromer) [inline notes.md]: Why parentheses?

Comment (ablwr) [reply notes.md]: I can't answer that; that's how it was in the original source.

Comment (retokromer) [reply notes.md]: And I would use `≈` instead of `= ~`

Comment (mbunkus) [reply notes.md]: :+1: to proper Unicode symbols.

Comment (ablwr) [reply notes.md]: Updated. Still going to wait until other nearby PRs are taken care of before resolving conflicts below.

File: notes.md [hunk 2]
```diff
@@ -129,114 +117,128 @@ As an example a file named `intro.mkv` could have a `SegmentUID` of `0xb16a58609
...
 
-    (1233 + 564264) * 1000000 = 565497000000
+# Multi-planar and 3D videos
 
-So, the Block in this example has a specific time of 565497000000 in nanoseconds. In milliseconds this would be 565497ms.
+There are 2 different ways to compress 3D videos: have each 'eye' track in a separate track and have one track have both 'eyes' combined inside (which is more efficient, compression-wise). Matroska supports both ways.
 
-# TimecodeScale Rounding
+For the single track variant, there is the StereoMode Element which defines how planes are assembled in the track (mono or left-right combined). Odd values of StereoMode means the left plane comes first for more convenient reading. The pixel count of the track (PixelWidth/PixelHeight) is the raw amount of pixels (for example 3840x1080 for full HD side by side) and the DisplayWidth/Height in pixels is the amount of pixels for one plane (1920x1080 for that full HD stream). Old stereo 3D were displayed using anaglyph (cyan and red colours separated). For compatibility with such movies, there is a value of the StereoMode that corresponds to AnaGlyph.
```
Comment (retokromer) [inline notes.md]: >(cyan and red colours separated)

I personally prefer the British `colours`, but American would be `colors`.

**Comment by mbunkus** [comment]
Same as the other PR; doesn't apply cleanly at the moment. Please rebase manually & push afterwards. Thanks.

**Comment by ablwr** [comment]
I'm waiting for the nearby notes.md issues to be resolved and then I'll rebase this one.

**Comment by mbunkus** [comment]
Those nearby changes have been merged.

**Comment by ablwr** [comment]
Urgh, I brought in commits from master. Gimme a minute.

**Comment by ablwr** [comment]
What a mess!!! Everything is out of sync, like bad timecodes! Shamefully closing this and opening something fresh. 😑

**Comment by ablwr** [comment]
BRB

**Pull request closed without merging**